lib/factory-helper/timestamps.rb

```diff
@@ -0,0 +1,31 @@
+module FactoryHelper
+  class Timestamps < Base
+
+    class << self
+      def between(from= ::Time.at(0), to= ::Time.local(2050, 12, 31,  23,  23,  59, 999_999.999))
+        random_time(from, to)
+      end
+
+      def forward(from= ::Time.at(0), seconds= 60)
+        to = from + seconds
+        random_time(from, to)
+      end
+
+      def backward(to= ::Time.at(0), seconds= 60)
+        from = to - seconds
+        random_time(from, to)
+      end
+
+    private
+
+      def random_time(from, to)
+        diff_time = to - from
+        dif_in_nsec = (diff_time * 1_000_000_000).to_i
+        rand_in_nsec = FactoryHelper::Config.random.rand(dif_in_nsec-1) + 1 # avoids 0
```

Comment:
to avoid using `FactoryHelper::Config.random.rand` here is a not so nice alternative:
``` ruby
      def randomize value
        result = value+1
        while result >= value   # needed because if value = 12_555_555, the block randomizes up to 12_999_999
          thousands = value.to_s.length/3
          remain = value / 1000**thousands
          result_arr = [(0..remain).to_a.sample(:random => FactoryHelper::Config.random) * (1000**thousands)]
          thousands.times do |index|
            result_arr << (0..1000).to_a.sample(:random => FactoryHelper::Config.random) * (1000**(thousands-index-1))
          end
          result = result_arr.inject(&:+)
        end
        result
      end
```